crates/aptos/src/lib.rs

```diff
@@ -43,6 +43,8 @@ pub enum Tool {
     #[clap(subcommand)]
     Move(move_tool::MoveTool),
     #[clap(subcommand)]
+    MultisigAccount(account::MultisigAccountTool),
```

Comment:
Let's rename this `Multisig` so the command will be `aptos multisig ...` rather than `aptos multisig-account`

Reply:
Updated